packages/typography/src/beta/components/Text.tsx

```diff
@@ -38,34 +38,43 @@ export type TextProps<C extends React.ElementType> = PolymorphicComponentProps<
   TextBaseProps
 >;
 
-const TypographyText = <C extends React.ElementType = 'p'>({
-  children,
-  as,
-  size,
-  variant,
-  weight = 'medium',
-  spacing,
-  className,
-  ...rest
-}: TextProps<C>) => {
-  const BodyElement = as || getSemanticTypeFromTextVariant(variant);
+const TypographyText = React.forwardRef<
+  HTMLElement,
+  TextProps<React.ElementType>
```

Comment:
Funker dette fortsatt med Polymorfisk type? Du sender ikke videre noe Generic (typ. C) her, så jeg tror kanskje ikke. Test i Typescript om du får typeforslag spesifikt til den du sender in som 'as'. 
(Er på mobil så får ikke testet akkurat nå)